src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -1,91 +1,85 @@
 package net.sf.jabref.importer.fileformat;
 
-import net.sf.jabref.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
 
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
 import net.sf.jabref.importer.OutputPrinterToNull;
 import net.sf.jabref.model.entry.BibEntry;
+
 import org.junit.Assert;
-import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
-import java.util.List;
-
 public class CopacImporterTest {
 
-    @Before
-    public void setUp() throws Exception {
-        if (Globals.prefs == null) {
-            Globals.prefs = JabRefPreferences.getInstance();
-        }
-    }
-
-    @Test
-    public void testIsRecognizedFormat() throws IOException {
-
-        CopacImporter importer = new CopacImporter();
-        try (InputStream stream1 = CopacImporterTest.class
-                .getResourceAsStream("CopacImporterTest1.txt"); InputStream stream2 = CopacImporterTest.class
-                .getResourceAsStream("CopacImporterTest2.txt"); InputStream stream3 = CopacImporterTest.class
-                .getResourceAsStream("IsiImporterTest1.isi"); InputStream stream4 = CopacImporterTest.class
-                .getResourceAsStream("IsiImporterTestInspec.isi"); InputStream stream5 = CopacImporterTest.class
-                .getResourceAsStream("IsiImporterTestWOS.isi"); InputStream stream6 = CopacImporterTest.class
-                .getResourceAsStream("IsiImporterTestMedline.isi")) {
-
-            Assert.assertTrue(importer.isRecognizedFormat(stream1));
+    private final String FILEFORMAT_PATH = "src/test/resources/net/sf/jabref/importer/fileformat";
 
-            Assert.assertTrue(importer.isRecognizedFormat(stream2));
 
-            Assert.assertFalse(importer.isRecognizedFormat(stream3));
+    /**
+     * Generates a List of all files in the package "/src/test/resources/net/sf/jabref/importer/fileformat"
+     * @return A list of Names
+     * @throws IOException
+     */
+    public List<String> getTestFiles() throws IOException {
+        List<String> files = new ArrayList<>();
+        Files.newDirectoryStream(Paths.get(FILEFORMAT_PATH)).forEach(n -> files.add(n.getFileName().toString()));
```

Comment:
a directory stream is a resource which must be closed. use a try-with-resources construct here, please.